auditors/capabilities/fixtures/capabilities-some-allowed.yml

```diff
@@ -21,18 +21,6 @@ spec:
             capabilities:
               add:
                 - SYS_TIME
-                - SYS_MODULE
+                - CHOWN
```

Comment:
This was intentionally not matching the override label since it's testing "some allowed"